src/Components/Body/ChatArea/ChatBody/ChatBody.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from "react";
+import { Message } from "./Message/Message";
+import { FixedSizeList } from "react-window";
+import "./ChatBody.css";
+
+interface messageBodyProps {
+  sender: string | null;
+  messages: string[] | undefined;
+}
+
+function ChatBody({ sender, messages }: messageBodyProps) {
+  const Row = useCallback(
+    ({ index, style }) => {
+      const message = messages ? messages[index] : undefined;
+      if (!message) {
+        return (null);
+      }
+      return (
+        <div style={style}>
+          {!message? null : (
+            <Message sender={sender} message={message} />
+          )}
+        </div>
+      );
+    },
+    [messages, sender]
+  );
+
+  return (
+    <div className="chat-body-container">
+      {messages &&
+        messages.map((message) => {
+          return <Message sender={sender} key={message} message={message} />;
```

Comment:
`message` can be repeated. Find a better key. Ideally each message should have a unique ID associated with it. You should use that for the key.